src/routes/group.route.ts

```diff
@@ -0,0 +1,27 @@
+import { Router } from 'express';
+import GroupsController from '@controllers/groupe.controller';
+import { CreateGroupDto } from '@dtos/group.dto';
+import { Routes } from '@interfaces/routes.interface';
+import validationMiddleware from '@middlewares/validation.middleware';
+
+class GroupsRoute implements Routes {
+  public path = '/groups';
+  public router = Router();
+  public groupsController = new GroupsController();
+
+  constructor() {
+    this.initializeRoutes();
+  }
+
```

Comment:
We are allowing any user to delete, create, update, and join. Don't we need an authentication and/or authorization middleware to verify the user? 

Please create a middleware to verify the user's role before allowing them to perform critical actions such as deleting a group.